scripts/automation/Radius/Functions/Private/Show-RadiusMainMenu.ps1

```diff
@@ -26,7 +26,7 @@ function Show-RadiusMainMenu {
         Write-Host "$([char]0x1b)[91m$($($expiringCerts.subject).Count) user certs will expire in 15 days `n"
     }
 
-    Write-Host "1: Press '1' to generate your Root Certificate."
+    Write-Host "1: Press '1' to generate or import your Root Certificate."
```

Comment:
Is this necessary here? I'm pretty sure we should just leave this as Press 1 to generate a root certificate